tests/test_adjoint_jacobian.py

```diff
@@ -583,8 +583,8 @@ def test_provide_starting_state(self, tol, dev):
 
         dM1 = dev.adjoint_jacobian(tape)
 
-        if device_name == "lightning.kokkos":
-            dev._pre_rotated_state = dev.state_vector  # necessary for lightning.kokkos
+        if device_name in ["lightning.kokkos", "lightning.qubit"]:
+            dev._pre_rotated_state = dev.state_vector
 
             qml.execute([tape], dev, None)
             dM2 = dev.adjoint_jacobian(tape, starting_state=dev._pre_rotated_state)
```

Comment:
I think we should remove the `_pre_rotated_state` attribute here.
```suggestion
        if device_name in ["lightning.kokkos", "lightning.qubit"]:
            qml.execute([tape], dev, None)
            dM2 = dev.adjoint_jacobian(tape, starting_state=dev.state_vector)
```

Reply:
Makes sense! Thank you!